src/status_im/subs/wallet/wallet_test.cljs

```diff
@@ -555,46 +555,46 @@
         (rf/sub [sub-name])))))
 
 (def chat-account
-  {:path                "m/43'/60'/1581'/0'/0"
-   :emoji               ""
-   :key-uid             "abc"
-   :address             "address-1"
-   :color-id            ""
-   :wallet              false
-   :name                "My Profile"
-   :type                "generated"
-   :chat                true
-   :customization-color :blue
-   :hidden              false
-   :removed             false})
+  {:path     "m/43'/60'/1581'/0'/0"
+   :emoji    ""
+   :key-uid  "abc"
+   :address  "address-1"
+   :color-id ""
+   :wallet   false
+   :name     "My Profile"
+   :type     "generated"
+   :chat     true
+   :color    :blue
```

Comment:
Just please make sure we are properly updating all the usages of this keyword

Reply:
Just the keypair-related views are using that key and it's changed now 👍

Reply:
Thanks for updating these 🙌